commit 3b3c9eccc57e8f32e707d4826588a4984c506c04 (refs/changes/77/3377/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-02-15T16:33:13-08:00 (1 year, 8 months ago)
    
    TRITON-151 node-sdc-clients vmapi tests are failing "wait for add nics using networks job ... Error: Timeout after 90 seconds"
    Reviewed by: Michael Zeller <mike.zeller@joyent.com>
    Approved by: Michael Zeller <mike.zeller@joyent.com>

diff --git a/test/vmapi.test.js b/test/vmapi.test.js
index 22faa7b..89816ce 100644
--- a/test/vmapi.test.js
+++ b/test/vmapi.test.js
@@ -284,8 +284,8 @@ test('vmapi', function (tt) {
 
             napi.listNetworks({ name: 'external' }, function (err2, nets2) {
                 t.ifError(err2);
-                t.ok(nets1);
-                EXTERNAL_NETWORK = nets1[0].uuid;
+                t.ok(nets2);
+                EXTERNAL_NETWORK = nets2[0].uuid;
                 t.end();
             });
         });
@@ -816,6 +816,7 @@ test('vmapi', function (tt) {
             t.ifError(err);
             t.ok(job);
             JOB_UUID = job.job_uuid;
+            t.ok(JOB_UUID, 'JOB_UUID is ' + JOB_UUID);
             t.end();
         });
     });
